tools/isobuild/package-source.js

```diff
@@ -1254,6 +1254,11 @@ Object.assign(PackageSource.prototype, {
       ...sourceReadOptions.exclude,
     );
 
+    if (process.env.METEOR_ANY_LEVEL_EXCLUDES) {
```

Comment:
Are you sure we want to do it this way? (by excluding files). There is a case to be made to always include both paths and differentiate purely in runtime. This way, it's easy to ensure that both paths compile and run. Easier to test also by temporarily altering runtime check without having to restart into a different mode.

Reply:
We wanted to be able to hide the naming from the code and do conditional compilation:
https://favro.com/organization/54f72cf77fb4776c31d006ee/2c21b1695c458efb5dcce5cf?card=Fav-155132

Also I made sure you can run both Favro and Planner8 at the same time with startdebug.